### ISSUE TITLE
Links with escaped spaces (%20) are reported as missing

### ISSUE THREAD

**Issue by henrikbarium** [microsoft/linkcheckermd#39]
```markdown
[Valid link](valid%20link.md)
```

Even if the file `valid link.md` exists, the above link is reported as broken.
